src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  display: flex;
+  justify-content: space-between;
+
+  .logo {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md?rgh-link-date=2022-03-08T16%3A16%3A55Z)

Reply:
i changed logo class to header__logo